src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -918,6 +920,13 @@ int fx_muxer_t::read_config_and_execute(
         roll_fwd_on_no_candidate_fx_val = pal::xtoi(roll_fwd_on_no_candidate_fx.c_str());
     }
 
+    pal::string_t additional_deps_serialized = additional_deps;
+    if (additional_deps_serialized.empty())
+    {
+        // additional_deps_serialized stays empty if DOTNET_ADDITIONAL_DEPS env var is not defined
+        pal::getenv(_X("DOTNET_ADDITIONAL_DEPS"), &additional_deps_serialized);
```

Comment:
EnvVar supports same semantic for specification as the command line. 

Also the general pattern is to have command line always override the envVar since it is an explicit action on part of the app.